Dockerfile

```diff
@@ -6,8 +6,7 @@ RUN make build
 FROM registry.ci.openshift.org/ocp/4.14:base
 LABEL io.k8s.display-name="openshift-state-metrics" \
       io.k8s.description="This is a component that exposes metrics about OpenShift objects." \
-      io.openshift.tags="OpenShift" \
-      maintainer="Haoran Wang <haowang@redhat.com>"
```

Comment:
May be this can be pointed to monitoring team
 
 `maintainer="OpenShift Monitoring Team <team-monitoring@redhat.com>"`

Reply:
thanks, updated